src/haddock/modules/analysis/seletopclusts/defaults.yaml

```diff
@@ -18,3 +18,16 @@ top_models:
   long: Number of best-ranked models to select per cluster. By default, 10 models are selected. If one expects to find many clusters (e.g., in the case of vague restraints), this number can be reduced to speed up the protocol. Instead, if few clusters are expected (very specific and localized restraints) this number can be increased.
   group: analysis
   explevel: easy
+method:
+  default: score
+  type: string
+  minchars: 2
+  maxchars: 15
+  choice:
+    - score
+    - size
+  title: Method used to define `best cluster`.
```

Comment:
for consistency with caprieval I would call this parameter "sortby" (see https://github.com/haddocking/haddock3/blob/cb4f1945ca7db4867426c79911f611a327a4ea00/src/haddock/modules/analysis/caprieval/defaults.yaml#L113C1-L113C7)

Reply:
OK nice